### PR TITLE
ADM-350 Fix Feedback about Pipeline tool section 

### PR DESCRIPTION
**Pull request by spadio9587**
## Summary
This branch is fix the error which mentioned in this #160 about pipeLineTool Section

### CODE REVIEW

File: frontend/__tests__/src/client/PipelineToolClient.test.ts [hunk 1]
```diff
@@ -1,48 +1,47 @@
...
   })
 
+  it('should throw error when pipelineTool verify response status 400', async () => {
+    server.use(rest.get(MOCK_PIPELINE_URL, (req, res, ctx) => res(ctx.status(400))))
+
+    pipelineToolClient.verifyPipelineTool(MOCK_PIPELINE_VERIFY_REQUEST_PARAMS).catch((e) => {
+      expect(e).toBeInstanceOf(Error)
+      expect((e as Error).message).toMatch(PIPELINE_TOOL_VERIFY_ERROR_MESSAGE[400])
+    })
+  })
```
Comment (LanxinWang) [inline frontend/__tests__/src/client/PipelineToolClient.test.ts:29]: need blank

File: frontend/__tests__/src/components/metrics/ConfigStep/PipelineTool.test.tsx [hunk 2]
```diff
@@ -1,17 +1,20 @@
-import { fireEvent, render, screen, waitFor, within } from '@testing-library/react'
+import { fireEvent, getByTestId, queryByText, render, screen, waitFor, within } from '@testing-library/react'
```
Comment (LanxinWang) [inline frontend/__tests__/src/components/metrics/ConfigStep/PipelineTool.test.tsx]: codacy error

File: frontend/src/components/Metrics/ConfigStep/MetricsTypeCheckbox/index.tsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+import { BOARD_TYPES, PIPELINE_TOOL_TYPES, REQUIRED_DATA_LIST } from '@src/constants'
+import { useState } from 'react'
 import { RequireDataSelections } from '@src/components/Metrics/ConfigStep/MetricsTypeCheckbox/style'
 import { Board } from '@src/components/Metrics/ConfigStep/Board'
 import { useAppDispatch, useAppSelector } from '@src/hooks/useAppDispatch'
-import { selectMetrics, updateBoard, updateMetrics } from '@src/context/config/configSlice'
+import { selectMetrics, updateBoard, updateMetrics, updatePipelineToolFields } from '@src/context/config/configSlice'
 import { PipelineTool } from '@src/components/Metrics/ConfigStep/PipelineTool'
 import { updateBoardVerifyState } from '@src/context/board/boardSlice'
+import { changePipelineToolVerifyState } from '@src/context/pipelineTool/pipelineToolSlice'
```
Comment (LanxinWang) [inline frontend/src/components/Metrics/ConfigStep/MetricsTypeCheckbox/index.tsx:10]: rename function updatexxx

**Review by LanxinWang — APPROVED** [review]
LGTM

**Review by guzhongren — APPROVED** [review]
LGTM

File: frontend/src/theme.ts [hunk 4]
```diff
@@ -44,3 +45,22 @@ export const theme = createTheme({
...
+export const VerifyButton = styled(Button)({
+  width: '3rem',
+  fontSize: '0.8rem',
+  fontWeight: '550',
+})
+export const ResetButton = styled(Button)({
+  width: '3rem',
+  fontSize: '0.8rem',
+  fontWeight: '550',
+  color: '#f44336',
```
Comment (fengxinhou) [inline frontend/src/theme.ts:65]: The style color of reset button may use the backgroundColor in theme

**Review by mengna-tw — APPROVED** [review]

**Pull request merged**